SQL/Archive/19.0/2018-01-26-default-inserting-is-null.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE mri_upload CHANGE `Inserting` `Inserting` tinyint(1) DEFAULT NULL;
```

Comment:
I ran this and no issue at all.
I am just posting this comment to make sure there are no cases where running a patch like this on databases where the column is already like that would fail?